addons/survey/models/survey_question.py

```diff
@@ -545,8 +563,33 @@ def _get_stats_data(self, user_input_lines):
             return table_data, [{'key': self.title, 'values': graph_data}]
         elif self.question_type == 'matrix':
             return self._get_stats_graph_data_matrix(user_input_lines)
+        elif self.question_type == 'scale':
+            table_data, graph_data = self._get_stats_data_scale(user_input_lines)
+            return table_data, [{'key': self.title, 'values': graph_data}]
         return [line for line in user_input_lines], []
 
+    def _get_stats_data_scale(self, user_input_lines):
```

Comment:
:socks: would define before '_get_stats_summary_data' (alphabetical ordering of stats methods)